docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+services:
+  codesniffer:
+    image: php:8.2.4-cli-alpine
```

Comment:
`composer.json` requires `"php": ">=7.4.0"` so probably need to adapt this image to 7.4, too. Might need to alter the composer 2.4 image as well, to something compatible with php 7.4.

This probably will be a good image for it: [7.4-cli-alpine3.16](
https://hub.docker.com/layers/library/php/7.4-cli-alpine3.16/images/sha256-1e1b3bb4ee1bcb039f559adb9a3fae391c87205ba239b619cdc239b78b7f2557?context=explore)

Reply:
updated to a cli version ^^^

Reply:
So, I'm wondering if I misread this. PHP >= is pretty broad. What is actually used in PRD? Is this package used in such a wide range that 7.4-8.2 is okay? I feel like CS will have different results based on the version of PHP, where 8.2 might have some stuff around nulls being passed to some functions that used to be allowed that aren't any more, and those might not get caught if we do change this. I guess I'm not sure the best way to handle that.

Reply:
I'm removing this as nothing is using the "codesniffer" service any longer.

Reply:
Prod in classic is on PHP 7.4.

php-api is on 8.2.4; that's defined in the repo in `Dockerfile.prod`.

The other repos typically have their PHP version defined in `docker-compose.yml` and it looks to be 8.1 or 8.2 across the board.

Reply:
This has been updated to run using the  7.4-cli-alpine3.16 version mentioned above. 

The codesniffer commands have been updated to run specifically on this image.